src/vstest.console/Processors/AeDebuggerArgumentProcessor.cs

```diff
@@ -123,29 +127,39 @@ private ArgumentProcessorResult InstallUnistallPostmortemDebugger(bool install)
             return ArgumentProcessorResult.Fail;
         }
 
-        // Validate ProcDumpToolDirectoryPath
-        if (!TryGetDirectoryInfo(_collectDumpParameters,
-            "ProcDumpToolDirectoryPath",
-            CommandLineResources.ProcDumpToolDirectoryPathArgumenNotFound,
-            CommandLineResources.InvalidProcDumpToolDirectoryPath,
-            out DirectoryInfo? procDumpToolDirectoryPath))
+        // Look for procdump
+        string? procDumpPath = null;
+        if (!TryGetDirectoryInfo(_collectDumpParameters, "ProcDumpToolDirectoryPath", out DirectoryInfo? procDumpToolDirectoryPath) &&
+            !new ProcDumpExecutableHelper(_processHelper, _fileHelper, _environment, _environmentVariableHelper).TryGetProcDumpExecutable(out procDumpPath)
```

Comment:
It's not changing much here but I would also new up the instance in ctor.

Reply:
done